src/components/core/card-filter/CardFilter.tsx

```diff
@@ -15,16 +15,19 @@ export interface CardFilterProps<T extends RequiredFields> {
   data: T[];
   tagsValueGetter: (data: T) => string[];
   valueGetter: (data: T) => ValueType;
+  inputField?: boolean;
```

Comment:
I would've expected `inputField` to be a component, not a boolean. Maybe rename to `inputFieldEnabled`.